pkg/metrics/metrics.go

```diff
@@ -11,27 +11,26 @@ import (
 const Async = "async"
 const Sync = "sync"
 const ImagePullTimeKey = "pull_duration_seconds"
-const ImageMountTimeKey = "mount_duration_seconds"
+const ImagePullTimeHistKey = "pull_duration_seconds_hist"
 const OperationErrorsCountKey = "operation_errors_total"
 
-var ImagePullTime = prometheus.NewHistogramVec(
+var ImagePullTimeHist = prometheus.NewHistogramVec(
 	prometheus.HistogramOpts{
 		Subsystem: "warm_metal",
-		Name:      ImagePullTimeKey,
+		Name:      ImagePullTimeHistKey,
 		Help:      "The time it took to pull an image",
-		Buckets:   []float64{0, 1, 5, 10, 15, 30, 60, 120, 180},
+		Buckets:   []float64{1, 5, 10, 15, 30, 60, 120, 300, 600, 900},
 	},
-	[]string{"operation_type"},
+	[]string{"error"},
 )
 
-var ImageMountTime = prometheus.NewHistogramVec(
-	prometheus.HistogramOpts{
+var ImagePullTime = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
 		Subsystem: "warm_metal",
-		Name:      ImageMountTimeKey,
+		Name:      ImagePullTimeKey,
 		Help:      "The time it took to mount an image",
-		Buckets:   []float64{0, 1, 5, 10, 15, 30, 60, 120, 180},
 	},
-	[]string{"operation_type"},
```

Comment:
same as above. additionally, reporting based on image and result will be more useful. go routine is launched to delete the value after 1m so we wont have values building up. included this change since some of the specific metrics had to change anyway with this implementation change.